### PR TITLE
OLH-1896-cache-our-frontends-static-assets-in-cloudfront

### PR DESCRIPTION
**Pull request by latifcabinetoffice**
## Proposed changes
[OLH-1896] Cloud front static caching on /assets/* and /public/*

### What changed
Addition of cache behaviours, enabling of caching static content in /assets/* and /public/*

### Why did it change
Improve performance of front end content loading

### Related links
N/A

## Checklists
N/A

### Environment variables or secrets
- [x] No environment variables or secrets were added or changed

## Testing

- Validated deployment in dev successfully updates cloud front config

[OLH-1896]: https://govukverify.atlassian.net/browse/OLH-1896?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by aenciso** [comment]
It might be worth waiting until this PR is merged https://github.com/govuk-one-login/di-account-management-backend/pull/260 which changes the flag we're using

**Comment by latifcabinetoffice** [comment]
> @latifcabinetoffice It might be worth waiting until this PR is merged [govuk-one-login/di-account-management-backend#260](https://github.com/govuk-one-login/di-account-management-backend/pull/260) which changes the flag from `IsNotDevelopment` to `IsNotDevelopmentOrDemo`

@aenciso Do you know if that change will be happening in the front end as well?

**Review by alex9smith — APPROVED** [review]

**Pull request merged**
